packages/graph/hash_graph/lib/graph/src/knowledge/entity/query.rs

```diff
@@ -211,7 +211,7 @@ pub enum EntityQueryPath<'q> {
     ///
     /// [`Entity`]: crate::knowledge::Entity
     /// [`LinkEntityMetadata::left_entity_id()`]: crate::knowledge::LinkEntityMetadata::left_entity_id
-    LeftEntity(Option<Box<Self>>),
+    LeftEntity(Box<Self>),
     /// Corresponds to the entity specified by [`LinkEntityMetadata::right_entity_id()`].
     ///
     /// It's `None` if this entity is not a link.
```

Comment:
I did remember us removing the `Option`s in the other PR. We should remove this line :)